robotiq_hande_driver/CMakeLists.txt

```
@@ -7,7 +7,19 @@ endif()
 
 find_package(ament_cmake REQUIRED)
 
```

Investigate this approach:
```suggestion
find_package(ament_cmake REQUIRED)
find_package(libmodbus REQUIRED)
```

---

resolved in: [9910a79](https://github.com/AGH-CEAI/robotiq_hande_driver/pull/2/commits/9910a79d76995127f366313f25583fcec30c32db)